Assets/MixedRealityToolkit.Tests/MixedReality.Toolkit.Tests.nuspec

```diff
@@ -14,14 +14,15 @@
     <releaseNotes>$releaseNotes$</releaseNotes>
     <tags>Unity MixedReality MixedRealityToolkit MRTK</tags>
     <dependencies>
+      <dependency id="Microsoft.MixedReality.Toolkit.Tests.Utilities" version="$version$" />
       <dependency id="Microsoft.MixedReality.Toolkit.Foundation" version="$version$" />
     </dependencies>
     <contentFiles>
       <files include="any\any\.PkgSrc\**" buildAction="None" copyToOutput="false" />
     </contentFiles>
   </metadata>
   <files>
-    <file src="**" exclude="*.nuspec;*.nuspec.meta;*.props;*.props.meta;*.targets;*.targets.meta" target="MRTK\" />
+    <file src="**" exclude="*.nuspec;*.nuspec.meta;*.props;*.props.meta;*.targets;*.targets.meta; TestUtilities\**" target="MRTK\" />
```

Comment:
Nit: Add a comment about why TestUtilities is being excluded (being that we have another package that is including those)

Reply:
Added